packages/preview/iconic-salmon-fa/1.0.0/examples/examples.typ

```diff
@@ -0,0 +1,104 @@
+#import "../iconic-salmon-fa.typ": *
```

Comment:
This import should use the package specification:
```suggestion
#import "@preview/iconic-salmon-fa:1.0.0": *
```

Reply:
Is there any hard requirement for this? I see that several other packages also reference the local version (which btw is the only way to access a new version to generate a new PDF file before publishing)

Reply:
No, it is not a hard requirement. It is just better because then people can copy/paste your example and start from here. It also makes it easier to tell which version of the package is used if the file is given out-of-context.
I agree that this is however not ideal to generate PDF files. The best solution as far as I know is to generate them with the relative import, and then update it to use the package import before submitting here.

Reply:
That's why I have included the instructions in the `README.md` :smile:

Reply:
Ok, I'll merge it as is then.